src/commands/mod.rs

```diff
@@ -289,6 +291,48 @@ impl DaemonControl {
         GetAddressResult::new(address)
     }
 
+    /// list addresses
+    pub fn list_addresses(&self, start_index: u32, count: u32) -> GetAddressesResult {
```

Comment:
not sure if start_index should be u32 or ChildNumber type

Reply:
Since `count` needs to be `u32` i think it's better to keep `start_index` as a `u32` too.